admin_guide/cluster_management.rst

```diff
@@ -487,23 +493,85 @@ To set non-sensitive libpq connection parameters to be used for all node connect
 
 .. code-block:: postgresql
 
   -- only superusers can access this table
 
+  -- add a password for user jdoe
   INSERT INTO pg_dist_authinfo
     (nodeid, rolename, authinfo)
   VALUES
     (123, 'jdoe', 'password=abc123');
 
 After this INSERT, any query needing to connect to node 123 as the user jdoe will use the supplied password. The documentation for :ref:`pg_dist_authinfo <pg_dist_authinfo>` has more info.
 
+.. code-block:: postgresql
+
+  -- update user jdoe to use certificate authenticaion
+  UPDATE pg_dist_authinfo
+  WHERE nodeid = 123 AND rolename = 'jdoe'
+  SET authinfo = 'sslcert=/path/to/user.crt sslkey=/path/to/user.key';
+
+This changes the user from using a password to use a certificate and keyfile while connecting to node 123 instead. Make sure the user certificate is signed by a certificate that is trusted by the worker you are connecting to and authentication settings on the worker allow for certificate based authentication. Full documentation on how to use client certificates can be found in `the postgres libpq documentation <https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT>`_.
+
+Changing ``pg_dist_authinof`` does not force any existing connection to reconnect.
+
+Setup Certificate Authority signed certificates
+-----------------------------------------------
+
+This section assumes you have a trusted Certificate Authority that can issue server certificates to you for all nodes in your cluster. It is recommended to work with the security department in your organisation to prevent key material be handled incorrectly. This guide only covers Citus specific configuration that needs to be applied, no best practices on PKI management.
+
+For all nodes in the cluster you need to get a valid certificate signed by the *same Certificate Authority*. The following **machine specific** files are assumed to be available on every machine:
+ * ``/path/to/server.key``: Server Private Key
+ * ``/path/to/server.crt``: Server Certifiacte or Certificate Chain for Server Key, signed by trusted Certifiacte Authority.
+
+Next to these machine specific files you need these cluster or CA wide files available:
+ * ``/path/to/ca.crt``: Certificate of the Certificate Authority
+ * ``/path/to/ca.crl``: Certificate Revocation List of the Certificate Authority
+
+.. note::
+
+   The Certificate Revocation List is likely to change over time. Work with your security department on a mechanism to get up to date versions of the revocation list on to all nodes in the cluster in a timely manner. A reload of every node in the cluster is required after the revocation list has been updated.
+
+Once all files are in place on the nodes the following settings need to be configured in the Postgres configuration file:
```

Comment:
```suggestion
Once all files are in place on the nodes the following settings need to be configured in the Postgres configuration file (PostgreSQL needs to be restarted for these settings to be applied):
```

Reply:
I don't think postgres requires a restart. I believe ssl changes are reloaded in postgres since pg10. I think adding a reload is required makes sense.

Reply:
Makes sense, I updated my commit suggestion accordingly.

Reply:
I have more generally phrased it and put it below the config block instead. Do you think it is ok like that now?

Reply:
I added a small comment updating that wording. I think with that it's okay.